<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wg_rs</class>
 <widget class="QWidget" name="wg_rs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gBox_rs">
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	border: 1px solid rgb(90, 90, 90);
    border-radius: 8px;
	margin-top: 9px;
}

QGroupBox:title{
	color: white;
	subcontrol-position: top center;
	padding: 0 5px;
	margin-top: -12px;
}

QGroupBox:title:disabled{
	color: gray;
	subcontrol-position: top center;
	padding: 0 5px;
	margin-top: -12px;
}</string>
     </property>
     <property name="title">
      <string>V-Ray Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,0,1">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>13</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cBox_renderSettings">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>High</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Min Subdivs:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_minSubdiv">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1,0,0">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Max Subdivs:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_maxSubdiv">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Shading rate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_shading">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1,0,0">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Noise threshold:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dsBox_noise">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Bucket size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_bucket">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="cBox_lights">
          <property name="text">
           <string>Lights</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_gi">
          <property name="text">
           <string>GI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_shadows">
          <property name="text">
           <string>Shadows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_displacement">
          <property name="text">
           <string>Displacement</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
